### PR TITLE
Lest use 0.0.0.0 instead of localhost, it will allow to use the server from external resources

### PR DESCRIPTION
**Pull request by oivoodoo**

### CODE REVIEW

**Comment by jaimeMF** [comment]
It should be an option, not the default.

**Comment by oivoodoo** [comment]
@jaimeMF ok. good point. I will change it by args.host. is it ok?

**Comment by jaimeMF** [comment]
Thanks! I have squashed and cherry picked the changes in d4b7cb25ace3aa807873db880c6571ff24412654

File: youtube_dl_server/server.py [hunk 1]
```diff
@@ -24,6 +24,15 @@ def main():
                         help=port_help
                         )
 
+    default_host = 'localhost'
+    host_help = 'The host the server will use. The default is: {}'
+    host_help = port_help.format(default_host)
+    parser.add_argument('-h', '--host',
```
Comment (jaimeMF) [inline youtube_dl_server/server.py]: `-h` conflicts with the `--help` option, so I have removed it.

Comment (oivoodoo) [inline youtube_dl_server/server.py]: @jaimeMF sorry, I missed it. :) thank you!

**Pull request closed without merging**